tardis/io/atomic.py

```diff
@@ -38,270 +37,106 @@ def tests_data_path(fname):
-
-    zeta_data = h5_file['zeta_data']
-    t_rads = zeta_data.attrs['t_rad']
-    return pd.DataFrame(zeta_data[:,2:], index=pd.MultiIndex.from_arrays(zeta_data[:,:2].transpose().astype(int)),
-                 columns=t_rads)
-
-
-def read_collision_data(fname):
-    if fname is None:
-        raise ValueError('fname can not be "None" when trying to use NebularAtom')
-
-    if not os.path.exists(fname):
-        raise IOError('HDF5 File doesn\'t exist')
-
-    h5_file = h5py.File(fname, 'r')
-
-    if 'collision_data' not in h5_file.keys():
-        raise ValueError('collision_data not available in this HDF5-data file. It can not be used with NLTE')
-
-    collision_data = np.array(h5_file['collision_data'])
-    collision_temperatures = h5_file['collision_data'].attrs['temperatures']
-
-    return collision_data, collision_temperatures
-
-
-def read_ion_cx_data(fname):
-    try:
-        h5_file = h5py.File(fname, 'r')
-        ion_cx_th_data = h5_file['ionization_cx_threshold']
-        ion_cx_sp_data = h5_file['ionization_cx_support']
-        return ion_cx_th_data, ion_cx_sp_data
-    except IOError, err:
-        print(err.errno)
-        print(err)
-        logger.critical('Cannot import. Error opening the file to read ionization_cx')
-
-
-def read_macro_atom_data(fname):
-    if fname is None:
-        raise ValueError('fname can not be "None" when trying to use NebularAtom')
-
-    if not os.path.exists(fname):
-        raise IOError('HDF5 File doesn\'t exist')
-
-    h5_file = h5py.File(fname, 'r')
-
-    if 'macro_atom_data' not in h5_file.keys():
-        raise ValueError('Macro Atom Data (macro_atom_data) is not in this HDF5-data file. '
-                         'It is needed for complex line interaction')
-    macro_atom_data = h5_file['macro_atom_data']
-
-    macro_atom_counts = h5_file['macro_atom_references']
-
-    return macro_atom_data, macro_atom_counts
-
-
 class AtomData(object):
     """
     Class for storing atomic data
 
-    AtomData
-    ---------
-
     Parameters
     ----------
-
-    basic_atom_data : `~astropy.table.Table`
-        containing the basic atom data: z, symbol, and mass
-
-    ionization_data : ~astropy.table.Table
-        containing the ionization data: z, ion, and ionization energy
-        ::important to note here is that ion describes the final ion state
-            e.g. H I - H II is described with ion=2
-
-    levels : ~astropy.table.Table
-        containing the levels data: z, ion, level_number, energy, g
-
-    lines : ~astropy.table.Table
-        containing the lines data: wavelength, z, ion, levels_number_lower,
-        levels_number_upper, f_lu, f_ul
-
-    macro_atom_data : tuple of ~astropy.table.Table
-        default ~None, a tuple of the macro-atom data and macro-atom references
-
-    zeta_data : ~dict of interpolation objects
-        default ~None
+    atom_data: pandas.DataFrame
```

Comment:
@wkerzendorf @yeganer I used the legacy names but I suggest changing `atom_data` -> `atomic_masses`, `ionization_data` -> `ionization_energies` (and I used these names in carsus). Not sure about other dataframes.